src/ray/protobuf/gcs.proto

```diff
@@ -310,6 +310,9 @@ message WorkerTableData {
   // The exception thrown in creation task. This field is set if this worker died because
   // of exception thrown in actor's creation task. Only applies when is_alive=false.
   RayException creation_task_exception = 18;
+  // The id of the worker.
+  bytes worker_id = 19;
```

Comment:
nit: this is already covered by `worker_address.worker_id`

Reply:
I thought it'd be more consistent (and clear) this is in the top message. But I have no strong opinion to just use that id inside another message.

Reply:
I would prefer no duplication here if possible so we eliminate the possibility that they are inconsistent.

Reply:
Sure. I can fix this part!